src/config.ini

```diff
@@ -1,5 +1,8 @@
 [Commands]
-adventure="adventure"
-farm="chop"
+adventure="adv"
+work="chainsaw"
 hunt="hunt"
-training="training"
\ No newline at end of file
+training="tr"
+lootbox="ed lb"
+farm="farm carrot"
```

Comment:
For the default value, this command would sent as `rpg farm farm carrot`. Is it correct?

Reply:
No, what would be sent would be `rpg farm carrot`. I have added a few more variables to the original config.ini file in order to accommodate these changes. Here is the code which explains it.
```
IniRead, farm_str, %A_ScriptDir%\config.ini, Commands, farm

Farm_RPG:
    ControlFocus,, ahk_exe Discord.exe
    ControlSend,, rpg farm, ahk_exe Discord.exe
    return

FarmC_RPG:
    ControlFocus,, ahk_exe Discord.exe
    ControlSend,, rpg %farm_str%, ahk_exe Discord.exe
    return
```

Reply:
Oh my bad!